app/controllers/healthcare_messaging_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+class HealthcareMessagingController < ApplicationController
+  # FIXME: when ID.me is working we need to use it here, but for now skip
+  #   and just rely on the http basic authentication.
+  skip_before_action :authenticate
+  before_action :authenticate_client
+  include ActionController::Serialization
+
+  ######################################################################################################################
+  ## MHV_CONFIG
+  ## Instantiates a new configuration used in establishing a session.
+  ######################################################################################################################
+  MHV_CONFIG = VAHealthcareMessaging::Configuration.new(
+    host: ENV['MHV_SM_HOST'],
+    app_token: ENV['MHV_SM_APP_TOKEN'],
+    enforce_ssl: Rails.env.production?
+  ).freeze
+
+  ######################################################################################################################
+  ## Pagingation constants
+  ######################################################################################################################
```

Comment:
and certain things like this don't really need to be commented -- should be descriptive enough as constants.